csharp/src/Ice/Internal/ConnectionFactory.cs

```diff
@@ -1504,6 +1518,9 @@ public void connectionStartFailed(Ice.ConnectionI connection, Ice.LocalException
         _state = StateHolding;
         _acceptorStarted = false;
 
+        // TODO: add MaxConnections property.
```

Comment:
I am confused, you already added this property to ObjectAdapter.

Reply:
I don't think I did but I'll add it.